src/stories/create-dataset/AuthorFormGroup.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { StoryObj, Meta } from '@storybook/react'
+import { WithLayout } from '../WithLayout'
+import { WithI18next } from '../WithI18next'
+import { AuthorFormGroup } from '../../sections/create-dataset/AuthorFormGroup'
+import { SubmissionStatus } from '../../sections/create-dataset/useCreateDatasetForm'
+import { initialState } from '../../sections/create-dataset/useDatasetValidator'
+import { DatasetMetadataSubField } from '../../dataset/domain/models/Dataset'
+
+const meta: Meta<typeof AuthorFormGroup> = {
+  title: 'Sections/Create Dataset Page/AuthorFormGroup',
+  component: AuthorFormGroup,
+  decorators: [WithI18next, WithLayout]
+}
+export default meta
+type Story = StoryObj<typeof AuthorFormGroup>
+
+export const Default: Story = {
```

Comment:
This is how I see the field. I think `Author Name ?` should be in the same line. Also the add button is not aligned with the field.

![image](https://github.com/IQSS/dataverse-frontend/assets/23359572/201a86e2-f79b-4a30-8fd1-8f2d3f80449e)

Reply:
thanks, I fixed the alignment problems